go/chat/convloader.go

```diff
@@ -113,16 +113,17 @@ type BackgroundConvLoader struct {
 	activeLoadCtx      context.Context
 	activeLoadCancelFn context.CancelFunc
 	suspendCount       int
+	storage            *storage.Storage
 
 	// for testing, make this and can check conv load successes
-	loads                 chan chat1.ConversationID
+	loadTestCh            chan chat1.ConversationID
 	testingNameInfoSource types.NameInfoSource
 	appStateCh            chan struct{}
 }
 
 var _ types.ConvLoader = (*BackgroundConvLoader)(nil)
 
-func NewBackgroundConvLoader(g *globals.Context) *BackgroundConvLoader {
+func NewBackgroundConvLoader(g *globals.Context, storage *storage.Storage) *BackgroundConvLoader {
```

Comment:
I'd like it if this guy didn't require a storage, see my comment below about the location of that `Queue` function.